lib/probes/cassandra-driver.js

```diff
@@ -85,8 +83,10 @@ module.exports = function (cassandra) {
     patchSendOnConnection(RequestHandler, sendOnConnection, connection)
   }
 
-  if (cassandra.Client && cassandra.Client.prototype) {
+  if (cassandra.Client && cassandra.Client.prototype && false) {
```

Comment:
`&& false` temporary?

Reply:
you found the bug i just fixed! the tests found it. yes, it was temporary (or supposed to be!) in order to test the `logMissing()` function i'd added. all probes need to log when they can't find something they need to patch. previously i'd just hardcoded a separate message for each error but it became a pattern. so rather than duplicate the code and have many unique messages `index.js` now has a `makeLogMissing()` function that returns a function that can be used by each probe.